src/games/strategy/triplea/ai/proAI/ProBidAI.java

```diff
@@ -1797,6 +1802,9 @@ private static float determineEnemyBlitzStrength(final Territory blitzHere, fina
    */
   private static List<Unit> findPlaneAttackersThatCanLand(final Territory start, final int maxDistance,
       final PlayerID player, final GameData data, final List<Territory> ignore, final List<Territory> checked) {
+    if (checked.isEmpty()) {
+      return new ArrayList<>();
+    }
```

Comment:
Seems unnecessary to even call this method when we know the return value already. I think it would be better to fix this by updating the caller code. Private method should not need to do null/empty checks generally because we should already know the answer.